packages/data/src/redux-store/metadata/test/selectors.js

```diff
@@ -1,117 +1,329 @@
 /**
- * External dependencies
+ * WordPress dependencies
  */
-import EquivalentKeyMap from 'equivalent-key-map';
+import { createRegistry } from '@wordpress/data';
 
-/**
- * Internal dependencies
- */
-import {
-	getIsResolving,
-	hasStartedResolution,
-	hasFinishedResolution,
-	isResolving,
-} from '../selectors';
+jest.useRealTimers();
+jest.setTimeout( 1000000 );
```

Comment:
Any idea why we need this? I believe maybe you forgot to delete this 😅 ?

Reply:
I did forget to remove it – I needed it to use a breakpoint :D removed now, thanks